pages/linux/pngcheck.md

```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Useful forensics tool for validating the integrity of image files.
+> Can also extract embedded images and text from the file.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Verify the integrity of an image file:
+
+`pngcheck {{image}}`
```

Comment:
Are we talking about images like the created with the `dd` command? Perhaps this better
```suggestion
`pngcheck {{file.img}}`
```

Reply:
Thank you for the feedback.  I hadn't thought about the ambiguity of the term.  It's actually for PNG image files.  I have updated the description to include (.png | .jng | .mng) image types and changed file.png in the examples.